packages/panels/src/components/sidebar-menu/SidebarMenuLink.tsx

```diff
@@ -15,11 +19,17 @@ export interface SidebarMenuLinkProps
       | "rightIcon"
       | "success"
     > {
+  renderLink?: SidebarLinkRenderer;
+  width?: CSSProperties["width"];
   selected?: boolean;
   indent?: boolean;
 }
 
-const selectedBorder = "4px solid var(--swui-sidebar-menu-item-selected)";
+export type SidebarLinkRenderer = (props: RenderLinkProps) => ReactNode;
```

Comment:
This is just a component?

```suggestion
export type SidebarLinkComponent = React.FC<RenderLinkProps>;
```

Reply:
VFC?

Reply:
it could be a component I guess but I imagined it to be a `renderLink` callback rather than a `RenderLink`-component prop?

Reply:
Yeah, they are very similar. Maybe not call the argument `props`?
But, technically, they are the same thing :)